### ISSUE TITLE
Lint for when you export an identifier twice

### ISSUE THREAD

**Issue by Hixie** [dart-lang/linter#263]
``` dart
export 'package:flutter/rendering.dart' show
    Axis,
    BoxConstraints,
    CrossAxisAlignment,
    CustomClipper,
    CustomPainter,
    FixedColumnCountGridDelegate,
    Axis,
    ViewportDimensionsChangeCallback;
```

...did not cause an error (note the duplicate `Axis`).

**Comment by srawlins** [comment]
I think this should even be a hint. I don't think there would be any false positives.

**Comment by pq** [comment]
@bwilkerson : should we move this over to the SDK repo?

**Comment by bwilkerson** [comment]
Yes.

**Comment by pq** [comment]
This issue was moved to dart-lang/sdk#33182
